### ISSUE TITLE
Pre-trained model for SR task

### ISSUE THREAD

**Issue by Zheng222** [roimehrez/contextualLoss#18]
@roimehrez Hello, Could you provide the pre-trained model for SR task or release the results of Set5, Set14, B100, PIRM_Val datasets?

**Comment by roimehrez** [comment]
All the results are available here
https://roimehrez.github.io/
search for set5

**Comment by roimehrez** [comment]
You can also find the PIRM test set results here

https://pirm.github.io/
